src/tfvc/scm/model.ts

```diff
@@ -174,8 +171,8 @@ export class Model implements Disposable {
                 if (_.contains(this._explicitlyExcluded, resource.resourceUri.fsPath.toLowerCase())) {
                     return excluded.push(resource);
                 }
-                //Versioned changes should always be included
-                if (resource.IsVersioned) {
+                //Versioned changes should always be included (as long as they're not deletes)
+                if (resource.IsVersioned && !resource.HasStatus(Status.DELETE)) {
```

Comment:
If it's the only file in the Included section, yes.  Basically, it's a 'detected change' and the check-in fails with this:

[Checkin] No changes were matched by any arguments.

Presumably, if there are other files in that Included list, those would get checked in.  Without this change, if the file is versioned and they delete it via the command prompt or explorer, it would appear in the Included list but not be checked in.  With this change, it'll be _excluded_ and when they move it to included, we'll run `tf delete` on it.